Description: handle mktemp as other mandatory commands
 For consistency, mktemp might be treated as other
 mandatory commands.
Origin: debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2014-08-06

--- a/sbin/firehol.in
+++ b/sbin/firehol.in
@@ -206,6 +206,7 @@
 which_cmd TAIL_CMD tail
 which_cmd LSMOD_CMD lsmod
 which_cmd MKDIR_CMD mkdir
+which_cmd MKTEMP_CMD mktemp
 which_cmd MV_CMD mv
 which_cmd RM_CMD rm
 which_cmd SED_CMD sed
@@ -404,7 +405,7 @@
 # Directories and files
 
 # Create an empty temporary directory we need for this run.
-if ! FIREHOL_DIR="`mktemp -d -t firehol-XXXXXXXXXX`"
+if ! FIREHOL_DIR="`${MKTEMP_CMD} -d -t firehol-XXXXXXXXXX`"
 then
             echo >&2
             echo >&2
--- a/configure.ac
+++ b/configure.ac
@@ -67,6 +67,7 @@
 AX_NEED_PROG([LOGGER_CMD], [logger], [logger])
 AX_NEED_PROG([LSMOD_CMD], [lsmod], [lsmod], [$PATH:/sbin:/usr/sbin])
 AX_NEED_PROG([MKDIR_CMD], [mkdir], [mkdir])
+AX_NEED_PROG([MKTEMP_CMD], [mktemp], [mktemp])
 AX_NEED_PROG([MV_CMD], [mv], [mv])
 AC_CHECK_PROG([PAGER_CMD], [pager], [pager])
 AC_CHECK_PROG([PAGER_CMD], [less], [less])
